pkgs/tools/misc/whatsapp-chat-exporter/default.nix

```
@@ -0,0 +1,38 @@
+{ lib, python3Packages, fetchFromGitHub
+
+, withEncryptedAndroid ? false
+, withCrypt12 ? false
+, withCrypt14 ? false
+, withCrypt15 ? false
+}:
+
+let version = "0.9.1";
+in python3Packages.buildPythonPackage {
+  pname = "whatsapp-chat-exporter";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "KnugiHK";
+    repo = "Whatsapp-Chat-Exporter";
+    rev = "refs/tags/${version}";
+    sha256 = "sha256-DvCYMfR9GgdP9rVpcoIR5nG9b4ToOBMG1a9OTvjfIiU=";
+  };
+
+  propagatedBuildInputs = with python3Packages;
+    ([ bleach jinja2 ] ++ (lib.optionals (withEncryptedAndroid || withCrypt15) [
+      pycryptodome
+      javaobj-py3
+    ]) ++ (lib.optionals (withCrypt12 || withCrypt14) [ pycryptodome ]));
```

```suggestion
  propagatedBuildInputs = with python3Packages; [
    bleach
    jinja2
  ] ++ lib.optionals (withEncryptedAndroid || withCrypt15) [
    pycryptodome
    javaobj-py3
  ] ++ lib.optionals (withCrypt12 || withCrypt14) [
    pycryptodome
  ];
```
One item per line here too.